gridstatus/caiso.py

```diff
@@ -647,6 +647,21 @@ def get_gas_prices(
         ]
         return df
 
+    def get_fuel_regions(self, verbose=False):
```

Comment:
Is it accurate to call this `get_fuel_regions`, or should it be something like `get_gas_pricing_regions`?

Reply:
CAISO calls these "fuel regions" so I will leave the name. At some point, we might want to rename `get_gas_prices` to `get_fuel_prices` but I'll leave as is for now.